File: app/src/main/java/com/example/musicwiki/repo/network/albumsinfo/Image.java

package com.example.musicwiki.repo.network.albumsinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Image {

    @SerializedName("size")
    @Expose
    private String size;
    @SerializedName("#text")
    @Expose
    private String text;

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

}

File: app/src/main/java/com/example/musicwiki/repo/network/toptracks/Image.kt
package com.example.musicwiki.repo.network.toptracks

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Image (
    @SerializedName("#text")
    @Expose
    var text: String? = null,

    @SerializedName("size")
    @Expose
    var size: String? = null
)